harness/nbjunit/src/org/netbeans/junit/ParametricTestSuite.java

```diff
@@ -36,23 +36,28 @@ public ParametricTestSuite() {
     
     /**
      * Constructs a ParametricTestSuite with the given name.
+     * @param name name of the suite
      */
     public ParametricTestSuite(String name){
         super(name);
     }
     
     /**
      * Returns an array of testcases for the given parameter.
+     * @param parameter parametre to retrieve the test case frome
+     * @return array of testcases
      */
     protected abstract ParametricTestCase[] cases(Object parameter);
     /**
      * Returns an array of parameters for this suite.
+     * @return array or parameter
      */
     protected abstract Object[] getParameters();
     
     /**
      * Factory method returns a new instance of a testcases.
      * Overrides the basic method so that it's needless any more.
+     * @return test case
```

Comment:
i gotta say I hate this kind of doc :)

This isn't adding any information it is just auto generated noise. Is there no way to turn off the warnings for things like this? This doesn't actually make the doc any better.

Reply:
I don't know how to be more specific for the warning, the return are not critical and I hate writing that too. But as some other warning prevents generation of javadoc I prefer having this noise than having half of apidoc.

Reply:
missing doc warnings should be harmless as far as I know. They shouldn't cause doc to be not generated at all - at least I never saw that happening before.

Reply:
you have some that are armless like return, some generate plain text instead of links on non existing page (for wrong @link / @see) some. and some break the build in later jdk.
but I can close my eyes on return :p

Reply:
And for everything else there's `doclint:none` :eyes:

Reply:
**doclint:html,reference,syntax** would be perfect I guess for the needs of having a apidoc in good shape.

Reply:
I put an additionnal commit to set doclint to verify only what html/reference/syntax. Thanks for the tip, I do prefer focus on links/href/html as it break website :p. Bad text for a parameter/return can be fixed later if people have concern on that.
--
I had to relax the failonwarning because javadoc on jdk18 create warning on depreciate with removal of some classes.